models/articles.model.js

```diff
@@ -32,3 +32,17 @@ exports.checkArticleExists = (article_id) => {
       }
     });
 };
+
+exports.updateArticleByArticleID = (article_id, inc_votes) => {
+  return db
+    .query(
+      `UPDATE articles SET votes=(SELECT votes FROM articles WHERE article_id=$1)+$2 WHERE article_id=$1 RETURNING *`,
+      [article_id, inc_votes]
+    )
+    .then(({ rows: articles }) => {
+      if (articles.length === 0) {
+        return Promise.reject({ status: 404, msg: "No article found" });
+      }
+      return articles[0];
```

Comment:
You're saying that this is all articles being returned, but I'm not sure that that's what it is. I think the naming convention here needs to be rethought :)

Reply:
updated